src/types/global.d.ts

```diff
@@ -16,4 +16,14 @@ declare module '*.svg' {
     export default content;
 }
 
+declare global {
```

Comment:
There is already a `src/types/modules/electron.d.ts` file which adds `electron` to `Window` interface. Could we use it?